test-data/unit/parse-errors.test

```diff
@@ -273,17 +273,8 @@ file:3: error: Syntax error in type comment
 file:3: error: Inconsistent use of "*" in function signature
 file:3: error: Inconsistent use of "**" in function signature
 
-[case testPrintStatementInPython35]
-# flags: --python-version 3.5
-print 1
-[out]
-file:2: error: Missing parentheses in call to 'print'
-
-[case testPrintStatementInPython37]
-# flags: --python-version 3.7
-print 1
-[out]
-file:2: error: Missing parentheses in call to 'print'. Did you mean print(1)?
+[case testPrintStatementInPython37Plus]
+print 1  # E: Missing parentheses in call to 'print'. Did you mean print(1)?
```

Comment:
```suggestion
print 1
[out]
file:1: error: Missing parentheses in call to 'print'. Did you mean print(1)?
```

Reply:
Although this shows a much bigger issue: The whole `ParseErrorSuite` is always skipped as we don't test with Python 3.7 anymore.
```
pytest mypy/test/testparse.py::ParseErrorSuite
```
https://github.com/python/mypy/blob/a538cc98d54031f25e44787a90649ea909877f12/mypy/test/testparse.py#L81-L82

Reply:
@cdce8p no, they are executed. Please see this https://github.com/python/mypy/actions/runs/5552756350/jobs/10140489051 with `ParseErrorSuite` failures.